CHANGELOG.md

```diff
@@ -1,12 +1,222 @@
 # Changelog
 
-## Unreleased
+## v0.4.0 - 2025-03-21
```

Comment:
The other releases link to the tag. Should this be linked?

Reply:
Oh yes. I suppose I'll need to go back and add the link after going through the release process and creating the tag, which is dependent on this PR.